src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -816,12 +816,14 @@ private void ImportProcedure(string library, ProcedureNode proc)
                             defaultArgumentNode = binaryExpr.RightNode;
                         }
                     }
+                    string shortName = null;
                     if (defaultArgumentNode != null)
                     {
+                        shortName = defaultArgumentNode.ToString();
```

Comment:
It finally depends on our long term strategy for default argument expressions - whether we intend to enforce developers to fully qualify their class names or not.